src/libraries/Microsoft.Extensions.Logging.Console/src/JsonConsoleFormatter.cs

```diff
@@ -16,6 +19,7 @@ namespace Microsoft.Extensions.Logging.Console
     internal class JsonConsoleFormatter : ConsoleFormatter, IDisposable
     {
         private IDisposable _optionsReloadToken;
+        private char[] _singleCharArray = new char[1];
```

Comment:
The Formatter should exists for the whole application lifetime so I don't think there is a need to get this from the ArrayPool